src/web/components/SlotBodyEnd.tsx

```diff
@@ -16,60 +22,77 @@ type Props = {
 
 export const SlotBodyEnd = ({
+    isSignedIn,
+    countryCode,
     contentType,
     sectionName,
     shouldHideReaderRevenue,
     isMinuteArticle,
     isPaidContent,
     tags,
 }: Props) => {
-    // Putting together the request payload
-    const contributionsPayload = {
-        tracking: {
-            ophanPageId: window?.guardian.config.ophan.pageViewId,
-            ophanComponentId: 'ACQUISITIONS_EPIC',
-            platformId: 'GUARDIAN_WEB',
-            campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
-            abTestName: 'remote_epic_test',
-            abTestVariant: 'api',
-            referrerUrl: window?.location.origin + window?.location.pathname
-        },
-        localisation: {
-            countryCode: 'GB', // TODO: make this dynamic
-        },
-        targeting: {
-            contentType,
-            sectionName,
-            shouldHideReaderRevenue,
-            isMinuteArticle,
-            isPaidContent,
-            tags,
-        },
-    };
+    const [epicContent, setEpicContent] = useState<{ html: string; css: string}>();
 
-    const endpointUrl = 'https://contributions.guardianapis.com/epic';
-    const { data: responseBody, error } = useApi(endpointUrl, {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(contributionsPayload),
-    });
+    useEffect(() => {
+        const callFetch = async () => {
+            // Putting together the request payload
+            const contributionsPayload = {
+                tracking: {
+                    ophanPageId: window?.guardian.config.ophan.pageViewId,
+                    ophanComponentId: 'ACQUISITIONS_EPIC',
+                    platformId: 'GUARDIAN_WEB',
+                    campaignCode: 'gdnwb_copts_memco_remote_epic_test_api',
+                    abTestName: 'remote_epic_test',
+                    abTestVariant: 'api',
+                    referrerUrl: window?.location.origin + window?.location.pathname
+                },
+                localisation: {
+                    countryCode
+                },
+                targeting: {
+                    contentType,
+                    sectionName,
+                    shouldHideReaderRevenue,
+                    isMinuteArticle,
+                    isPaidContent,
+                    tags,
+                    showSupportMessaging: shouldShowSupportMessaging(),
+                    isRecurringContributor: isRecurringContributor(isSignedIn),
+                    lastOneOffContributionDate: getLastOneOffContributionDate(),
+                },
+            };
 
-    if (error) {
-        window.guardian.modules.sentry.reportError(error, 'slot-body-end');
-        return null;
-    }
-
-    if (responseBody && responseBody.data) {
-        const { data } = responseBody;
+            const getEpicContent = () => {
+                const endpointUrl = 'https://contributions.guardianapis.com/epic';
```

Comment:
No need for this to be its own variable IMO

Reply:
Yeah, I agree, I much prefer reading code with as much stuff inline as possible

Reply:
> No need for this to be its own variable IMO

Only useful cause we often toggle between local and PROD urls so have this on a separate line to comment/uncomment more easily... but otherwise you're right.

Reply:
When we support that toggling then we can make that change :) again its a minor change so if you want to keep im happy

Reply:
Updated to have the api url line with the fetch call